scripts/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "outDir": "../dist/dev-infra-scripts",
     "target": "es2015",
     "lib": ["es2016"],
+    "moduleResolution": "node",
```

Comment:
This was necessary, because vscode was flagging any imports to `node_modules` dependencies as invalid.